roles/backup/tasks/init.yml

```
@@ -0,0 +1,61 @@
+---
+
+- name: Delete any existing management pod
+  community.kubernetes.k8s:
```

```suggestion
  k8s:
```

Please remove any FQCNs.